src/ImmersedBoundaries/partial_cell_bottom.jl

```diff
@@ -129,15 +138,14 @@ end
     # Get node at face above and defining nodes on c,c,f
     z = znode(i, j, k+1, underlying_grid, c, c, f)
 
-    # Get bottom height and fractional Δz parameter
-    h = @inbounds ib.bottom_height[i, j, 1]
-    ϵ = ibg.immersed_boundary.minimum_fractional_cell_height
+    # Get bottom z-coordinate and fractional Δz parameter
+    zb = @inbounds ib.bottom_height[i, j, 1]
 
     # Are we in a bottom cell?
     at_the_bottom = bottom_cell(i, j, k, ibg)
 
-    full_Δz = Δzᶜᶜᶜ(i, j, k, ibg.underlying_grid)
-    partial_Δz = max(ϵ * full_Δz, z - h)
+    full_Δz    = Δzᶜᶜᶜ(i, j, k, ibg.underlying_grid)
+    partial_Δz = z - zb
```

Comment:
This should go in a different PR.

Reply:
The new numerical bottom height is located at by convention at `z⁻ + Δz * (1 - ϵ)` so `ϵ * full_Δz` == `z - h` removing the necessity of the max. I thought this change can clean up the partial cell code

Reply:
actually, there is a bug here with how the immersed bottom height is computed. I ll correct it

Reply:
i just dont understand how this is related to a PR that implements a new function called `static_column_depth`